readme.md

```diff
@@ -282,6 +283,13 @@ The following properties are defined:
 * `providedServices`: array of supported services as listed [in the EKAP documentation](https://www.vdv.de/431-2sdes-v-1-3.pdfx#page=51)
 * `requestLimitPolicy`: request-policy in the [markup of the `RateLimit` HTTP header](https://www.ietf.org/archive/id/draft-ietf-httpapi-ratelimit-headers-04.html#name-expressing-rate-limit-polic)
 
+#### MOTIS
+
+The following properties are defined:
+* `endpoint`: Base URL for the API
+* `motisVersion`: version of the [motis-server](https://github.com/motis-project/motis/releases)
+* `loadedDatasets`: dictionary of loaded timetables with an identifier-string for the dataset and the according timespan
```

Comment:
Does MOTIS provide this timespan (of validity I assume?) via its API? If it does, I would like to omit it here, because the information would likely end up being outdated.

Reply:
The covered time span of the loaded timetable is available via `GET /lookup/schedule_info` (example: https://europe.motis-project.de/lookup/schedule_info).

Reply:
Then I would omit it here as well. As the geographic coverage is already defined as a general property and the time span can be retrieved from the api, I would remove the loadedDatasets property completely, as it does not seem needed anymore.